packages/core/content-manager/admin/src/pages/EditView/components/FormInputs/DynamicZone/ComponentCategory.tsx

```diff
@@ -59,15 +60,23 @@ const ComponentCategory = ({
             </ComponentBox>
           ))}
         </Grid>
-      </Accordion.Content>
+      </ResponsiveAccordionContent>
     </Accordion.Item>
   );
 };
 
+const ResponsiveAccordionContent = styled(Accordion.Content)`
+  container-type: inline-size;
+`;
+
 const Grid = styled(Box)`
   display: grid;
-  grid-template-columns: repeat(auto-fit, 14rem);
+  grid-template-columns: repeat(auto-fill, 100%);
   grid-gap: ${({ theme }) => theme.spaces[1]};
+
+  @container (min-width: ${() => RESPONSIVE_CONTAINER_BREAKPOINTS.sm}) {
```

Comment:
no need for a function

Reply:
I get an error here without a function 🤔 

```
Cannot access 'RESPONSIVE_CONTAINER_BREAKPOINTS' before initialization
```